languages/ruby/reference/exercise-concepts/nucleotide-count.md

```diff
@@ -0,0 +1,44 @@
+# Concepts of nucleotide-count
+
+## General
+
+- class methods: used as the starting point of the exercise. They can be named with `self.[method_name]` notation or with a `self` block.
```

Comment:
Not sure where to put this, but class methods can also be created by extending a module... modules are mentioned in the next bullet.  How to express this?

Reply:
Do you think it's better to group all of them as class methods instead of having two concepts? Something like:
```
- class methods: used as the starting point of the exercise. They can be named with `self.[method_name]` notation or with a `self` block. Another way of creating class methods is extending a module.
```

Reply:
One way that I often express this is by stating _class level methods_ because of the different ways to bring them to that point.  So I think that works.